client/lib/directly/test/index.js

```diff
@@ -96,22 +80,20 @@ describe( 'index', () => {
 		const name = 'Richie Rich';
 		const email = 'richie@richenterprises.biz';
 
-		it( 'does nothing if Directly hasn\'t been initialized', () => {
+		it( 'initializes Directly if it hasn\'t already been initialized', () => {
```

Comment:
a very reasonable place to use the backtick for strings to avoid the escaping around the single quote

```js
`initialized Directly if it hasn't already been initialized`
```